Source/SPDataImport.m

```diff
@@ -308,12 +308,37 @@ - (void)importFile
 		}
 
 		if (importFileName == nil) return;
+		
+		// Check to see if current connection has existing tables, if so warn
+		if([[tablesListInstance tables] count] > 1){
```

Comment:
I believe this needs spacer before and after `()`

Reply:
Is this a style thing?

Reply:
Since I don't control the PR, I'm gonna merge it but you can add the spaces @Kaspik and PR the tweak if desired :)